packages/app-builder/src/components/Workflows/WorkflowPayloadEvaluationNode.tsx

```diff
@@ -0,0 +1,354 @@
+import { type AstNode, NewUndefinedAstNode } from '@app-builder/models';
+import {
+  isKnownOperandAstNode,
+  isMainAstBinaryNode,
+  isMainAstNode,
+  isMainAstUnaryNode,
+} from '@app-builder/models/astNode/builder-ast-node';
+import {
+  isBinaryMainAstOperatorFunction,
+  isUnaryMainAstOperatorFunction,
+} from '@app-builder/models/astNode/builder-ast-node-node-operator';
+import { isDataAccessorAstNode } from '@app-builder/models/astNode/data-accessor';
+import { getDataAccessorAstNodeField } from '@app-builder/services/ast-node/getDataAccessorAstNodeField';
+import { getAtPath, getParentPath, parsePath } from '@app-builder/utils/tree';
+import { AstBuilderDataSharpFactory } from '@ast-builder/Provider';
+import { NodeTypeError } from '@ast-builder/styles/NodeTypeError';
+import { computed } from '@preact/signals-react';
+import { memo, type ReactNode, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import invariant from 'tiny-invariant';
+import { match } from 'ts-pattern';
+import { MenuCommand } from 'ui-design-system';
+import { Icon } from 'ui-icons';
+
+import { EditionAstBuilderOperand } from '../AstBuilder/edition/EditionOperand';
+import { getErrorsForNode } from '../AstBuilder/edition/helpers';
+import { AstBuilderNodeSharpFactory } from '../AstBuilder/edition/node-store';
+import { OperatorSelect, type OperatorSelectOptions } from '../AstBuilder/edition/OperatorSelect';
+
+function useSiblings(stringPath: string) {
+  const rootNode = AstBuilderNodeSharpFactory.useSharp().value.node;
+
+  const path = parsePath(stringPath);
+  const parentPath = getParentPath(path);
+  if (!parentPath || parentPath.childPathSegment?.type !== 'children') {
+    return [];
+  }
+  const childIndex = parentPath.childPathSegment.index;
+  const parentNode = getAtPath(rootNode, parentPath.path);
+  if (!parentNode || parentNode.name !== '=') {
+    return [];
+  }
+
+  return [
+    ...parentNode.children.slice(0, childIndex),
+    ...parentNode.children.slice(childIndex + 1),
+  ];
+}
+
+// Filtered operators for workflow payload evaluation
+// Include basic comparison, string matching, and list operations
+const workflowOperatorFunctionsOptions: OperatorSelectOptions<string> = {
+  '=': {},
+  '≠': { keywords: ['!='] },
+  '<': {},
+  '<=': {},
+  '>': {},
+  '>=': {},
+  IsInList: {},
+  IsNotInList: {},
+  StringContains: {},
+  StringNotContain: {},
+  StringStartsWith: {},
+  StringEndsWith: {},
+  ContainsAnyOf: {},
+  ContainsNoneOf: {},
+  IsEmpty: {},
+  IsNotEmpty: {},
+};
+
+export const WorkflowPayloadEvaluationNode = memo(function WorkflowPayloadEvaluationNode(props: {
```

Comment:
I don't understand why you basically recreated a for of the ASTBuilder, were there something you had a hard time doing? Maybe that why editing the case title template node is erroring.